recipes/libtiff/all/conanfile.py

```diff
@@ -9,27 +9,31 @@ class LibtiffConan(ConanFile):
     license = "MIT"
     homepage = "http://www.simplesystems.org/libtiff"
     topics = ("tiff", "image", "bigtiff", "tagged-image-file-format")
-    exports_sources = ["CMakeLists.txt"]
+    exports_sources = "CMakeLists.txt", "patches/**"
     generators = "cmake", "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False],
-               "fPIC": [True, False],
-               "lzma": [True, False],
-               "jpeg": [True, False],
-               "zlib": [True, False],
-               "zstd": [True, False],
-               "jbig": [True, False],
-               "webp": [True, False],
-               "cxx":  [True, False]}
-    default_options = {"shared": False,
-                       "fPIC": True,
-                       "lzma": True,
-                       "jpeg": True,
-                       "zlib": True,
-                       "zstd": True,
-                       "jbig": True,
-                       "webp": True,
-                       "cxx":  True}
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "lzma": [True, False],
+        "jpeg": [False, "libjpeg-turbo", "libjpeg"],
```

Comment:
What will happen to consumers who were forcing `jpeg=True`?

Reply:
This will fail with this pr.
The options should also be renamed to `with_*`.

Maybe we should start with a deprecation process warning users settings these options that in approx 1 year the option will be removed.

Reply:
@madebr it will be better add with_ now and add a deprecation warning for jpeg option, just like @SpaceIm did for Jasper: https://github.com/conan-io/conan-center-index/blob/a812152aab4b8d093aba6637e25732345e80f32a/recipes/jasper/all/conanfile.py#L51 At least the users will have an information about this new important change.